installer/roles/oc-cluster-up/defaults/main.yml

```diff
@@ -7,6 +7,8 @@ hawkular_metrics: "false"
 
 host_config_dir: "{{playbook_dir}}/../ui"
 
-cluster_version: latest
+cluster_image: feedhenry/origin
```

Comment:
is that a temporary thing?

Reply:
for now yes cause all the things latest appear to be broken